HW_1/zadanie16.py

```diff
@@ -0,0 +1,24 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+# Последняя строка содержит число X.
+
+#*Пример:*
+
+#5
+ #   7 -2 3 5 1
+  #  3
+   # -> 1
+
+size = int(input("Введите количество жлементов массива N: "))
+some_list = list(range(1,size))
+print(some_list)
+
+numberX = int(input("Введите число X:"))
+sumX = 0
+
+for i in range(0, len(some_list)):
```

Comment:
Во-первых, ноль указывать не обязательно, если им начинается с нуля. Во-вторых, у вас уже есть параметр, который содержит длину списка.
```suggestion
for i in range(n):
```
И в третьих, можно проходиться напрямую по элементам списка, не используя функцию `range`